plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -789,7 +794,7 @@ private function maybe_display_feature_incompatibility_warning(): void {
 
 		// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
 		?>
-		<div class="notice notice-warning">
+		<div class="notice notice-error">
```

Comment:
Function mentions showing a warning, but class indicates an error 🤔 It seems that either function name and description should be renamed or we should revert class name to show a warning?